README.md

```diff
@@ -479,4 +481,40 @@ In this example `OPENPBTA_PATHSIG=0.75` species an environment variable `OPENPBT
 Any environment variables prefixed with `OPENPBTA_` are passed to the specified shell script.
 Environment variables without this prefix are not passed.
 
+### Molecular-subtyping 
+
+If you would like to identify molecular subtype membership for new RNAseq PBTA samples belonging to the following broad_histology
+ * [`molecular-subtyping-EWS`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-EWS)
+ * [`molecular-subtyping-HGG`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-HGG)
+ * [`molecular-subtyping-LGAT`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-LGAT)
+ * [`molecular-subtyping-embryonal`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-embryonal)
+ * [`molecular-subtyping-CRANIO`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-CRANIO)
+ * [`molecular-subtyping-EPN`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-EPN)
+ * [`molecular-subtyping-MB`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-MB)
+ * [`molecular-subtyping-neurocytoma`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-neurocytoma)
+
+<!--TODO: Add WGS/WXS summarization modules.-->   
+
+```
+bash scripts/run-for-subtyping.sh
+```
+
+Running this will re-run RNA-seq specific summary file generation modules as well as molecular-subtyping-* modules to generate a `compiled_molecular_subtypes_with_clinical_pathology_feedback.tsv` with molecular_subtype which is updated by pathology/clinical feedback information when available.
```

Comment:
```suggestion
Running this will re-run RNA-seq specific summary file generation modules as well as molecular-subtyping-* modules to generate the `compiled_molecular_subtypes_with_clinical_pathology_feedback.tsv` file containing the `molecular_subtype` column.
```